src/spikeinterface/postprocessing/correlograms.py

```diff
@@ -295,19 +416,19 @@ def compute_correlograms_numba(sorting, window_size, bin_size):
 
     Implementation: Aurélien Wyngaard
     """
-
     assert HAVE_NUMBA, "numba version of this function requires installation of numba"
 
-    num_bins = 2 * int(window_size / bin_size)
+    num_bins, _ = _compute_num_bins(window_size, bin_size)
     num_units = len(sorting.unit_ids)
+
     spikes = sorting.to_spike_vector(concatenated=False)
     correlograms = np.zeros((num_units, num_units, num_bins), dtype=np.int64)
 
     for seg_index in range(sorting.get_num_segments()):
         spike_times = spikes[seg_index]["sample_index"]
         spike_labels = spikes[seg_index]["unit_index"]
 
-        _compute_correlograms_numba(
+        _compute_correlograms_numba_new(
             correlograms, spike_times.astype(np.int64), spike_labels.astype(np.int32), window_size, bin_size
```

Comment:
We should use `arr.astype(dtype, copy=False)`

Reply:
Thanks!